### DISCUSSION TITLE
打包 vercel 部署的应用，一直失败

### DISCUSSION THREAD

**Discussion by cs118651** [tw93/Pake#384] [General]
本地命令行和github action（deb & appimage）都试了，job 是 success 的状态，但是没有产物

<img width="1315" alt="image" src="https://user-images.githubusercontent.com/29137793/230944967-477d9d4e-b49a-4207-80dd-5d62e3724951.png">

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/29137793/230944843-5e01d888-544c-44f9-949c-ed106a617040.png">

**URL👉: https://chat-gpt-next-web-drab-gamma-54.vercel.app/**

**github action👉: https://github.com/cs118651/Pake/actions**

> 是使用 https://github.com/Yidadaa/ChatGPT-Next-Web 在 vercel 上构建的一个 gpt 应用

**Comment by Tlntin** [comment]
actions会自动安装最新的pake-cli，而 Pake-cli目前正在2.0beta中，目前存在一些bug，所以会导致失败，急用可以手动安装pake-cli 1.3.1稳定版。

可以再试试，已经修复了。目前已将beta包删了，actions会自动安装稳定版1.3.1。
actions日志：https://github.com/tw93/Pake/actions/runs/4662619810

**Reply by cs118651** [reply]
感谢，重新 run 已经有产物了，但是打开是空白页

actions日志：https://github.com/cs118651/Pake/actions/runs/4663196971
系统版本：mac os 12.0.1 (21A559) Intel 芯片

**Reply by Tlntin** [reply]
空白页这个和网络以及ssl证书有关了，可以多打开几次试试。还不行就只能等2.0出正式版再试了，2.0的tauri原生框架相对wry会稳定许多。

**Reply by Tlntin** [reply]
用新版给你打包了一个，你可以试试，如果还是空白，那我也没办法了。可能是应用不合适。
[output-macos-latest.zip.zip](https://github.com/tw93/Pake/files/11195957/output-macos-latest.zip.zip)

**Reply by cs118651** [reply]
这个直接连应用都打不开了 🤯，能安装、但点击无响应

**Reply by Tlntin** [reply]
再试试，actions已经默认是tauri 2.0 beta版了，说不定有改善。

**Reply by xyy002** [reply]
我也出现了这样的事 build成功了 但是没有安装包

**Reply by Tlntin** [reply]
> 我也出现了这样的事 build成功了 但是没有安装包

看你的日志，显然是命名不规范导致的。
![image](https://user-images.githubusercontent.com/28218658/233752691-0ce528be-4fb8-4899-ac5e-c3bd0b32c18e.png)

**Reply by xyy002** [reply]
> > 我也出现了这样的事 build成功了 但是没有安装包
> 
> 
> 
> 看你的日志，显然是命名不规范导致的。
> 
> ![image](https://user-images.githubusercontent.com/28218658/233752691-0ce528be-4fb8-4899-ac5e-c3bd0b32c18e.png)
> 
> 

啊哈 我重新试试 🙏

**Reply by nbwzx** [reply]
我也出现了这样的事 build成功了 但是没有安装包
https://github.com/nbwzx/Pake/actions/runs/4831518865/jobs/8609077572

**Reply by Tlntin** [reply]
> 我也出现了这样的事 build成功了 但是没有安装包 https://github.com/nbwzx/Pake/actions/runs/4831518865/jobs/8609077572

看你的日志貌似是字节源抽风了，明天我再修一下。

**Reply by tw93** [reply]
@nbwzx @xyy002 应该修复了试试看
